### PR TITLE
HDXDSYS-685 Output csvs

### PR DESCRIPTION
**Pull request by mcarans**
Works now (thanks to Alex)

### CODE REVIEW

File: dump_views.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+today=$(date +%Y%m%d-%H%M%S)
```
Comment (turnerm) [inline dump_views.sh:3]: Can you add some colons ;)

File: dump_views.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+mkdir -p csv_export
+
+for table in location admin1 admin2 population operational_presence humanitarian_needs
+do
+  echo "Saving $table"
+  docker exec -t postgres-container psql -U postgres -d hapi -c "\copy (select * from ${table}_view) TO STDOUT (FORMAT csv, DELIMITER ',',  HEADER);" | tee >(gzip > csv_export/${table}.csv.gz) >(md5sum > csv_export/${table}.hash) >/dev/null
+done
+
+docker exec -t postgres-container pg_dump -U postgres -Fc hapi -f hapi_db.pg_restore
+docker cp postgres-container:/hapi_db.pg_restore csv_export/hapi_db.pg_restore
```
Comment (turnerm) [inline dump_views.sh:16]: Should we keep the DB restore file in a separate directory on Github? It may get a bit lost in all of the csvs and hashes

**Review by turnerm — APPROVED** [review]
Looks great, thanks for fighting with docker!

File: dump_views.sh (hunk 1 above)
Comment (mcarans) [reply dump_views.sh:3]: Good idea!

File: dump_views.sh [hunk 3]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+today=$(date +%Y%m%d-%H%M%S)
+
+echo "Started at $today"
+
+mkdir -p csv_export
+
+for table in location admin1 admin2 population operational_presence humanitarian_needs
```
[the remark this answers is not in the document]
Comment (mcarans) [reply dump_views.sh]: Good idea! A bit tricky but I've done it

File: dump_views.sh (hunk 2 above)
Comment (mcarans) [reply dump_views.sh]: Yes I've done that now

File: dump_views.sh [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+do
+  echo "Saving $table"
+  docker exec -t postgres-container psql -U postgres -d hapi -c "\copy (select * from ${table}_view) TO STDOUT (FORMAT csv, DELIMITER ',',  HEADER);" | tee >(gzip > csv_export/${table}.csv.gz) >(md5sum > csv_export/${table}.hash) >/dev/null
```
[the remark this answers is not in the document]
Comment (mcarans) [reply dump_views.sh]: I've made more readable and stripped the dashes

**Pull request merged**